pkg/process/metadata/parser/service_test.go

```diff
@@ -213,6 +213,27 @@ func TestExtractServiceMetadata(t *testing.T) {
 			},
 			expectedServiceTag: "process_context:myservice-core",
 		},
+		{
+			name: "dotnet cmd with dll",
+			cmdline: []string{
+				"/usr/bin/dotnet", "./myservice.dll",
+			},
+			expectedServiceTag: "process_context:myservice",
+		},
+		{
+			name: "dotnet cmd with dll and options",
+			cmdline: []string{
+				"/usr/bin/dotnet", "-v", "--", "/app/lib/myservice.dll",
+			},
+			expectedServiceTag: "process_context:myservice",
+		},
+		{
+			name: "dotnet cmd with unrecognized options",
+			cmdline: []string{
+				"/usr/bin/dotnet", "run", "--project", "./projects/proj1/proj1.csproj",
+			},
+			expectedServiceTag: "process_context:",
```

Comment:
Do we want to add an empty tag in this case?

Reply:
I will double check but the dotnet cli can be used to perform a wide range of things (also to build project) and we do not want probably to surface a "service name" in this case

Reply:
I changed as suggested. Definitely makes sense to default like could be before. thanks